src/components/BounceRulesActivity/index.jsx

```diff
@@ -0,0 +1,198 @@
+import React from "react";
+import { CSVLink } from "react-csv";
+import "./index.scss";
+import Breadcrumb from "@sendgrid/ui-components/breadcrumb";
+import { StatefulTabs as Tabs, Tab } from "@sendgrid/ui-components/tabs";
+import Loader from "@sendgrid/ui-components/loader";
+import PropTypes from "prop-types";
+import { Redirect } from "react-router";
+import Header from "../Header";
+import { Row } from "../Row";
+import { Column } from "../Column";
+import RuleFilter from "./RuleFilter";
+import Pagination from "../Pagination";
+import EmptyRules from "./EmptyRules";
+import ActivityLogContainer from "./ActivityLogContainer";
+import { WriteSelectors } from "./selectors";
+import ActivityDetailsModal from "./ActivityDetailsModal";
+
+const BounceRulesContainer = ({
+  updateSearchToken,
```

Comment:
not necessary, but sometimes when lists like this get really long I alphabetize it to make it easier to find things later on